opslevel/datasource_opslevel_team.go

```diff
@@ -90,12 +90,10 @@ func (teamDataSource *TeamDataSource) Schema(ctx context.Context, req datasource
 		Attributes: map[string]schema.Attribute{
 			"alias": schema.StringAttribute{
 				MarkdownDescription: "The alias attached to the Team.",
-				Computed:            true,
```

Comment:
Why is `Computed:` being removed here and in the `"id"` attribute?

Reply:
@davidbloss my intention is to match what is in https://registry.terraform.io/providers/OpsLevel/opslevel/0.11.0/docs/data-sources/team (removing computed here so that the field is optional)

Reply:
Attributes can be both Computed and Optional. Not sure I understand this change